plugins/VersionUpgrade/VersionUpgrade44to45/VersionUpgrade44to45.py

```diff
@@ -75,8 +93,42 @@ def upgradeStack(self, serialized: str, filename: str) -> Tuple[List[str], List[
         # Update version number.
         if "metadata" not in parser:
             parser["metadata"] = {}
+
+        # Check for leftover hidden machines. They should be deleted along with their associated files
+        stack_id = parser["general"]["id"]
+        if parser["metadata"]["setting_version"] == "10" and stack_id is not None:
```

Comment:
This upgrade step is defined as upgrading from `setting_version` 10 to 11, so checking this is unnecessary.